src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import UserDetail from "../../components/UserDetail";
+import UserEmail from "../../components/UserEmail";
+import UserPassword from "../../components/UserPassword";
+import defaultProfileImage from "../../images/default-profile-image.png";
+import "./index.scss";
+
+export default function ProfilePage() {
+  const [elementSelected, setElementSelected] = React.useState("userDetail");
+
+  return (
+    <Container fluid="md" className="profileContainer">
+      <Row className="profilePortrait">
+        <h2>Profile</h2>
+        <div className="userPortrait">
+          <img src={defaultProfileImage} alt="User portrait" />
+        </div>
+      </Row>
+      <Row className="section userDetail">
+        <button
+          className="sectionTitle"
+          onClick={() => setElementSelected("userDetail")}
+          type="button"
+        >
+          <i
+            className={
+              elementSelected === "userDetail" ? "arrow down" : "arrow right"
+            }
+          />
+          User Details
```

Comment:
please add translation to all buttons

Reply:
I am leaving the translation to do it later actually.

For the margin, I will update the style after adding the components to it

Reply:
this is just a layout

Reply:
I see. I wish you mentioned it in PR description

Reply:
don't commit code that's "just layout", because when you commit halfway code its probably not going to get changed later. someone is going to forget. 

basically once you commit the code we are relying on you to remember to fix it later, and if you dont, you have a really poorly laid out page in your demo

Reply:
unless i'm misinterpreting this comment. what is missing from this PR?

if only translation is missing, thats ok

if anything styling is missing, then that's not ok

Reply:
this is a complete component. I meant by layout that this is a container to three other component as show in the screenshot. the translation is just missing. @louisrli

Reply:
I think you can at least add margin between header and profile image, since they won't be affected by you adding components in next PRs. other margins can be added after you add components